includes/options/class-amp-options-menu.php

```diff
@@ -132,7 +135,10 @@ public function add_menu_items() {
 	 * @since 1.0
 	 */
 	public function render_theme_support() {
-		$theme_support = AMP_Theme_Support::get_support_mode();
+		$theme_support = AMP_Options_Manager::get_option( Option::THEME_SUPPORT );
+
+		/** @var AmpSlugCustomizationWatcher $amp_slug_customization_watcher */
+		$amp_slug_customization_watcher = Services::get( 'amp_slug_customization_watcher' );
```

Comment:
This should be injected instead, provided that `AMP_Options_Menu` is instantiated via the injector.

Reply:
Agreed. Since `AMP_Options_Menu` is not yet instantiated that way, I used `Services` in the time being.